.gitignore

```diff
@@ -1 +1,4 @@
 /build
+DartConfiguration.tcl
+/CMakeFiles
+/CMakeCache.txt
```

Comment:
For these two directories, please remove the leading slash. This pattern only matches `${SRC_ROOT}/CMakeFiles` but not `${SRC_ROOT}/build/CMakeFiles`. Usually I use `mkdir build && cd build && cmake ..` to keep the build in its own directory, so it made sense to ignore the whole build directory in the source root. However, it seems like you are building in the source tree. Might as well just ignore all of the CMake generated files no matter where they are.

Reply:
You can actually replace this entirely with this: https://github.com/github/gitignore/blob/master/CMake.gitignore.

Reply:
Maybe add this and any generated files you can find: https://github.com/github/gitignore/blob/master/C.gitignore.